contracts/mocks/MockMargin.sol

```diff
@@ -50,4 +55,12 @@ contract MockMargin  {
     ) external returns (uint256[2] memory amounts) {
       IAmm(amm).swap(trader, inputToken, outputToken, inputAmount, outputAmount);
     }
+
+       function withdraw(
```

Comment:
does this function do anything? did you mean to remove it?

Reply:
just for test pass